Essentials/src/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -338,6 +342,17 @@ public void addFireworkMeta(final CommandSource sender, final boolean allowShort
         }
     }
 
+    private Color getColorFromRgbHexString(String color) {
+        Color c;
+        try {
+            c = Color.fromRGB(Integer.decode(color.substring(color.indexOf('[') + 4, color.indexOf(']') - 1)));
+        } catch (Exception e) {
```

Comment:
It's better to check for the exceptions that can happen rather than catch `Exception`.